projects/addon-editor/components/toolbar-tools/font-style/font-style.component.ts

```diff
@@ -15,8 +15,14 @@ import {map} from 'rxjs/operators';
     changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class TuiFontStyleComponent {
+    private toolsSet: Set<TuiEditorTool> = new Set(defaultEditorTools);
+
     @Input()
-    enabledTools: ReadonlyArray<TuiEditorTool> = defaultEditorTools;
+    set enabledTools(value: ReadonlyArray<TuiEditorTool> | Set<TuiEditorTool>) {
```

Comment:
`ReadonlyArray` is legacy, right? `Set` is also a valid input for `Set` constructor so you can just write `this.toolsSet = new Set(value)`.

Reply:
fixed